format/errors.js

```diff
@@ -0,0 +1,20 @@
+const { format } = require('logform');
+
+/*
+ * function errors (info)
+ * Handles special 'err' field and formats error according to ECS
+ */
+module.exports = format((info) => {
+  if (!Object.prototype.hasOwnProperty.call(info, 'err')) return info;
+
+  const { err, error, ...restOfInfo } = info;
```

Comment:
You're deconstructing `error` here but never actually using it...

Reply:
yes, I dont need it in the final obj returned. is there's other way to do it?

Reply:
oh, nevermind, I dont need to deconstruct it at all..